MatrixSDK/NotificationCenter/MXNotificationCenter.m

```diff
@@ -489,31 +524,24 @@ - (void)addRuleWithId:(NSString*)ruleId
                  kind:(MXPushRuleKind)kind
               pattern:(NSString *)pattern
            conditions:(NSArray<NSDictionary *> *)conditions
-                notify:(BOOL)notify
-                 sound:(BOOL)sound
-             highlight:(BOOL)highlight
+               notify:(BOOL)notify
+                sound:(BOOL)sound
+            highlight:(BOOL)highlight
+{
+    [self addRuleWithId:ruleId kind:kind pattern:pattern conditions:conditions notify:notify soundName:@"default" highlight:highlight];
```

Comment:
Should this be supplying `@"default"` even if `sound` is false?

Reply:
Oh you are dead right, that was a bit careless.